test/validation_test.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:test/test.dart';
+import 'package:web_scraper/validation.dart';
+
+void main() {
+  test('Error: use [http/https] protocol', () {
```

Comment:
add positive test messages for all tests in the file.
example: Checks for [http/https] protocol